src/App.scss

```diff
@@ -1,17 +1,38 @@
+@font-face {
+  font-family: "ms-sans-ser";
+  src: url(./fonts/ms-sans-ser-bold.ttf);
+}
+
 iframe {
   display: none;
 }
 
 html {
-  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  font-family: "Courier New", Courier, monospace;
+}
+
+body {
+  background-image: url(img/ms95.jpg);
```

Comment:
Consider using a more descriptive background image. `ms95.jpg` is not a descriptive name. 🖼️